scripts/firedrake-install

```diff
@@ -17,6 +17,7 @@ from itertools import chain
 import re
 
 osname = platform.uname()[0]
+arch   = platform.uname()[4]
```

Comment:
This is a linting failure (we prefer not to align `=` signs). I do not know why the CI system did not pick it up!

Can you run `make lint` in the root of the firedrake directory, I suspect it will report some errors which will want fixed. Thanks!

Reply:
done!